src/main/java/org/traccar/protocol/TopinProtocolDecoder.java

```diff
@@ -60,6 +63,21 @@ private void sendResponse(Channel channel, int length, int type, ByteBuf content
         }
     }
 
+    private void updateTime(Channel channel, int type){
```

Comment:
Just noticed another space missing here.

Reply:
Fixed in 1d7d66a.